templates/base.html

```diff
@@ -1,5 +1,6 @@
 <!doctype html>
-{% load static bootstrap3 %}
+{% load static from staticfiles %}
```

Comment:
What's the difference between load static and load static from staticfiles?

Reply:
https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#static
Changed in Django 1.10:

In older versions, you had to use {% load static from staticfiles %} in your template to serve files from the storage defined in STATICFILES_STORAGE. This is no longer required.

Reply:
good catch